src/REPLMode.jl

```diff
@@ -1275,13 +1276,15 @@ The `startup.jl` file is disabled during precompilation unless julia is started
     status
     status [-p|--project]
     status [-m|--manifest]
+    status [pkg...]
 
 Show the status of the current environment. By default, the full contents of
 the project file is summarized, showing what version each package is on and
 how it has changed since the last git commit (if in a git repo), as well as
 any changes to manifest packages not already listed. In `--project` mode, the
 status of the project file is summarized. In `--manifest` mode the output also
-includes the dependencies of explicitly added packages.
+includes the dependencies of explicitly added packages. Package names may be 
+given as arguments to limit the status output to those packages.
```

Comment:
Maybe: "If there are any packages listed as arguments the output will be limited to those packages." or something?